core/pouch/migrations.js

```diff
@@ -216,6 +216,31 @@ const migrations /*: Migration[] */ = [
         return doc
       })
     }
+  },
+  {
+    baseSchemaVersion: 7,
+    targetSchemaVersion: 8,
+    description: 'Use generated ids instead of string based on path',
```

Comment:
Why do you need a migration? We can keep the old generated ids, even if we use them as an opaque value now.

Reply:
Good idea. This would mean less records to migrate and less hassle to make sure they're all coherent.